docs/source/matlab/tenmat.rst

```diff
@@ -0,0 +1,22 @@
+``tenmat``
```

Comment:
I have a PR up now that updates the tenmat constructors. I believe this is correct currently but once Danny merges my PR this will go stale.